content/en/docs/demo/services/quote.md

```diff
@@ -92,8 +92,44 @@ $span->addEvent('Quote processed, response sent back', [
 
 ## Metrics
 
-TBD
+In this demo, metrics are emitted by the batch trace and logs processors. The
+metrics describe the internal state of the processor, such as number of exported
+spans/logs, the queue limit, and queue usage.
+
+You can enable metrics by setting the environment variable
+`OTEL_PHP_INTERNAL_METRICS_ENABLED=true`.
+
+A manual metric is also emitted, which counts the number of quotes generated,
+including attributes for the number of items and total cost.
+
+### Initializing metrics
+
+To generate metrics, you need to create an instrument such as a counter, gauge,
+or histogram. The following example retrieves a meter from the globally
+configured Meter Provider and uses it to create a counter:
+
+```php
+$counter = Globals::meterProvider()
+    ->getMeter('my-meter')
+    ->createCounter('my-counter');
+```
+
+Metrics accumulate and are exported periodically based on the value configured
+in `OTEL_METRIC_EXPORT_INTERVAL`.
+
+### Generate metrics
+
+An instrument provides a method to record a new value against it. This example
+shows how to `add` values to the counter:
+
+```php
+$counter->add(2);
+```
```

Comment:
I am not sure if this belongs here as this reads like generic documentation for metrics and not how it is done in the demo?

Reply:
I agree with @svrnm, in this page we should explain how we are doing in the demo itself:

```
        //manual metrics 
         static $counter; 
         $counter ??= Globals::meterProvider() 
             ->getMeter('quotes') 
             ->createCounter('quotes', 'quotes', 'number of quotes calculated'); 
         $counter->add(1, ['number_of_items' => $numberOfItems]);
```

Reply:
Have updated to refer to what the demo is doing.